atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/sweep/GetEmptyLatestValues.java

```diff
@@ -53,7 +53,7 @@ public GetEmptyLatestValues(
      */
     public Set<Cell> execute() {
         Set<Cell> result = Sets.newHashSet();
-        for (List<CellWithTimestamps> batch : Iterables.partition(cellTimestamps, batchSize)) {
+        for (List<CellWithTimestamps> batch : Lists.partition(cellTimestamps, batchSize)) {
```

Comment:
I've filed https://github.com/palantir/gradle-baseline/issues/621 so we can build this into an error-prone rule and hopefully catch other occurrences!